examples/human/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:24.04
```

Comment:
I'm duplicating this Dockerfile and `compose.yaml` in three places in this PR. Maybe I shouldn't do that!